.github/actions/setup-goversion/action.yaml

```diff
@@ -0,0 +1,13 @@
+# This action extracts the go version from Dockerfile and uses this version setup go
```

Comment:
wat?

```suggestion
# This action sets up go using the same version as in the Dockerfile
```

Reply:
I'm going to leave the first half as it's techincally true, but altered the last half of the sentence.